addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -30,6 +40,23 @@ function factory(dependencies) {
             this.update({ isOpen: false });
         }
 
+        async createMeetingChannel() {
+            const channelData = await this.env.services.rpc({
+                model: 'mail.channel',
+                method: 'create_group',
+                kwargs: {
+                    default_display_mode: 'video_full_screen',
+                    partners_to: [this.messaging.currentPartner.id],
+                },
+            });
+            const channel = this.messaging.models['mail.thread'].insert(
+                this.messaging.models['mail.thread'].convertData(channelData)
+            );
+            this.update({ mostRecentMeetingChannel: link(channel) });
+            this.channelInvitationForm.update({ doFocusOnSearchInput: true });
+            this.channelInvitationForm.searchPartnersToInvite();
```

Comment:
This is unrelated to "createMeetingChannel", wrong method (we can change it later)